### PR TITLE
coccinelle 1.0.8

### PR DESCRIPTION
**Pull request by Bo98**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/coccinelle.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-  url "http://coccinelle.lip6.fr/distrib/coccinelle-1.0.6.tgz"
-  sha256 "8452ed265c209dae99cbb33b67bc7912e72f8bca1e24f33f1a88ba3d7985e909"
-  revision 1
+  url "https://github.com/coccinelle/coccinelle.git",
+      :tag      => "1.0.8",
+      :revision => "d678c34afc0cfb479ad34f2225c57b1b8d3ebeae"
+  head "https://github.com/coccinelle/coccinelle.git"
 
   bottle do
     rebuild 1
```
Comment (SMillerDev) [inline Formula/coccinelle.rb]: can be removed

Comment (Bo98) [reply Formula/coccinelle.rb]: Don't we usually not touch bottle blocks?

Comment (SMillerDev) [reply Formula/coccinelle.rb]: The bottle block needs to stay, the `rebuild 1` can go. That's basically a `revision 1` when you don't want to update it for everyone.

Comment (Bo98) [reply Formula/coccinelle.rb]: BrewTestBot seems to handle it: https://github.com/BrewTestBot/homebrew-core/commit/51f1bb96f059c5e865420c31f76ea24c493c6251

Comment (SMillerDev) [reply Formula/coccinelle.rb]: Hmmm, TIL

**Pull request closed without merging**